authorize_error.go

```diff
@@ -66,7 +66,11 @@ func (f *Fosite) WriteAuthorizeError(rw http.ResponseWriter, ar AuthorizeRequest
 	query.Add("state", ar.GetState())
 
 	var redirectURIString string
-	if !(len(ar.GetResponseTypes()) == 0 || ar.GetResponseTypes().ExactOne("code")) && !errors.Is(err, ErrUnsupportedResponseType) {
+	if ar.GetRequestForm().Get("response_mode") == "form_post" {
```

Comment:
I think it would make sense to validate this response mode in the authorize handler! And then have `ar.ResponseMode() fosite.ResponseMode` which is an enum of `fosite.ResponseModePost` and `fosite.ResponseModeQuery` (default).

Reply:
+1, One small issue with setting default responseMode in the authorize handler. Depending on the flow the default  responseMode changes.

Reply:
Sorry. I missed it somehow. It is done now.